evap/student/templates/student_vote.html

```diff
@@ -285,7 +285,7 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                     }
                 }).catch(error => {
                     // show a warning if the post isn't successful
-                    document.getElementById('submit-error-warning').style.display = 'block';
+                    document.getElementById('submit-error-warning').classList.remove("d-none");
```

Comment:
The code before set display to `block`. Is this still the case if just `d-none` is removed?

Reply:
Yes, but now only implicitly/by default (since a paragraph element simply is a block element and thus has `display: block` by default). I initially considered this to be a win, not too sure about this now.